src/renderers/__tests__/EventPluginHub-test.js

```diff
@@ -24,6 +24,8 @@ describe('EventPluginHub', () => {
   });
 
   it('should prevent non-function listeners, at dispatch', () => {
+    // ReactDOMFiberComponent warns for non-function event listeners
+    spyOn(console, 'error');
```

Comment:
We should still add an expectation. So that it doesn't emit some other unwanted warning by mistake in the future.